.circleci/continue_config.yml

```diff
@@ -39,35 +61,43 @@ jobs:
           host: os01.eu-west-2.opensearch.localhost.localstack.cloud
       - restore_cache:
           keys:
-            - gradle-{{ checksum "<< pipeline.parameters.workflow-directory >>/build.gradle.kts" }}
+            - gradle-{{ checksum "<< parameters.project >>/build.gradle.kts" }}
             - gradle-
       - run:
-          command: ./gradlew :model:check :<< pipeline.parameters.workflow-directory >>:check
+          command: ./gradlew :<< parameters.project >>:check
       - save_cache:
           paths:
             - ~/.gradle
-          key: gradle-{{ checksum "<< pipeline.parameters.workflow-directory >>/build.gradle.kts" }}
+          key: gradle-{{ checksum "<< parameters.project >>/build.gradle.kts" }}
       - store_test_results:
-          path: build/test-results
+          path: << parameters.project >>/build/test-results
       - store_artifacts:
-          path: build/reports/tests
+          path: << parameters.project >>/build/reports/tests
 
 workflows:
   version: 2
-  build-test-and-deploy:
+  build-test-and-deploy-model:
+    when: << pipeline.parameters.build-model >>
+    jobs:
+      - validate-model:
+          filters:
+            tags:
+              ignore: /.*/
+  build-test-and-deploy-indexer:
```

Comment:
note that we end up with 3 separate workflows - one for model, one for search and one for indexer